pyproject.toml

```diff
@@ -39,8 +39,10 @@ module = [
   "botocore.*",
   "bson.*",
   "celery.*",
+  "citext.*",
   "dask.*",
   "deepdiff.*",
+  "defusedxml.ElementTree.*",
```

Comment:
This library doesn't have any type hints so I'm following the recommendation from mypy
https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-library-stubs-or-py-typed-marker